packages/commonwealth/client/scripts/views/components/component_kit/cw_wallets_list.tsx

```diff
@@ -308,7 +324,7 @@ export class CWWalletsList implements m.ClassComponent<WalletsListAttrs> {
                         // eslint-disable-next-line max-len
                         window.location.href = `https://app.axieinfinity.com/login/?src=commonwealth&stateId=${stateId}`;
                       } else {
-                        vnode.state.error(result.error || 'Could not login');
+                        console.log(result.error || 'Could not login');
```

Comment:
i think the vnode.state.error here was for the UI to show the error message. console log doesn't do that for us

Reply:
Unfortunately it was dead code, so it wasn't setting the error to anything (there's no `error` in `state`).